week_5/week_5_quiz.md

```diff
@@ -0,0 +1,118 @@
+# Week 5 Quiz
+
+### Quiz Instructions
+
+In this Week 5 Quiz, we will ask you a few questions about the ABCD neurocognitive assessments and FAIR data concepts.
+
+***
+
+### ABCD (neurocognitive assessments) Questions
+
+**Question 1**
+
+ What were the overarching motivations for the tasks selected for the neurocognitive battery?
+
+ - [ ] the tasks maximize sensitivity to development and other emergent factors across a decade of assessment, starting from the ages of 9-10 
+ - [ ]  the tasks minimize participant burden 
+ - [ ]  the tasks are comparable to those used for other large studies 
+ - [ ]  the tasks cover multiple domains
+ - [ ]  all the above
+
+**Question 2**
+
+This lecture covered the results of Thompson et al, 2019 whereby principal component analysis was applied to 12 neurocognitive assessments. Based on this study, what were the main principal components underlying the different tasks administered to participants at baseline? Select all that apply.
+
+- [ ] Reward 
+- [ ] General cognitive ability
+- [ ] Executive function
+- [ ] Learning and memory
+
+**Question 3**
+
+Which of these tasks are only offered at one timepoint? Select all that apply.
+
+- [ ] Picture Vocabulary
+- [ ] Rey Auditory Verbal Learning Test
+- [ ] Stanford Mental Arithmetic Response Time Eval
+- [ ] Dimensional Change Card Sort
+
+**Question 4**
+
+Which of these are true of the Cash Choice and Delay Discounting tasks? Select all that apply.
+
+- [ ] A strong association has been reported between temporal delay and substance use 
+- [ ] Both of these tasks are administered at each timepoint
+- [ ] The decision participants make regarding receiving $75 in three days or $115 in three months are developmentally stable 
```

Comment:
```suggestion
- [ ] The decision participants make about whether to receive $75 in three days or $115 in three months is developmentally stable 
```
Mostly "are" to "is", but the other proposed wording change makes it clearer, I think.